Rakefile

```diff
@@ -1,7 +1,5 @@
 require "yast/rake"
 
-Yast::Tasks.submit_to :sle12sp3
```

Comment:
looks like you use the wrong target branch. Do you really want to fix it in SLE12-SP3 ( aka leap 42.3 ) ?

Reply:
also some other changes looks wrong in this branch as it is from newer branch.